CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+#How to contribute
+TODO
+ 
+For the details on the Records Management project see[here.](https://community.alfresco.com/docs/DOC-6387-project-overview-records-management)
+
+##Getting started
+What you need to get started:
+
+* A [Jira](https://issues.alfresco.com/jira/projects/ALF/issues/?filter=allopenissues) account,
```

Comment:
Worth linking to the RM JIRA project: https://issues.alfresco.com/jira/projects/RM ?

Reply:
Possibly think it would be worth thinking about where would be the best place (for now I've added it in the additional info section)